src/utils/viewconf.js

```diff
@@ -176,6 +208,36 @@ export function updateViewConfigOnSelectGenomicInterval(currViewConfig, viewId,
     return newViewConfig;
 }
 
+/**
+ * Set the `selectRows` option for a particular track, and return the updated view config object.
+ * @param {object} currViewConfig A valid higlass view config object.
+ * @param {number[]} selectedRows The array of row indices, which will become the value of the track option.
+ * @param {string} neighborViewId The view ID for the track of interest.
+ * @returns {object} The new view config. 
+ */
+export function addViewConfigForNewTrack(currViewConfig, viewConfigToAdd, neighborViewId, position) {
```

Comment:
Can you add documentation comments for the `viewConfigToAdd` variable (or its renamed version) and the position variable?